tests/__snapshots__/template.test.js.snap

```diff
@@ -6,9 +6,9 @@ exports[`extends 1`] = `
  *
```

Comment:
@hosseinmd I just noticed this. It's this formatting of `@class` and the text below a bug?

Reply:
Yes this is a bug, but I can't fix. And i don't know why

Reply:
Ok. Just wanted to ask. I'll open an issue for it and try to fix it in the next few days.